Update README.md (#5)
vtex-apps/country-flags: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/country-flags) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # VTEX COUNTRY FLAGS

Reviewer: ```suggestion
# Country Flags
```

> ⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Country Flags app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> This component receives the country code and renders that respective country's flag in SVG.

Reviewer: ```suggestion
The Country Flags app provides flags in SVG icons to be displayed according to the countries' ISO code.
```

Reviewer: It would be nice if we could add a media (image/gif) showing the app once rendered!

## Configuration 

> 

Reviewer: In this section, you first must **add the primary instructions** that will allow users to use the app's blocks in their store, such as:

1. Adding the app as a theme dependency in the `manifest.json` file;
2. Declaring the app's main block in a given theme template or inside another block from the theme.

Remember to add a table with all blocks exported by the app and their descriptions. You can verify an example of it on the [Search Result documentation](https://vtex.io/docs/components/all/vtex.search-result@3.56.1/). 

Next, add the **props table** containing your block's props. 

If the app exports more than one block, create several tables - one for each block. For example:

### `block-1` props

| Prop name | Type | Description | Default value |
| :---------: | :----: | :----------: | :------------: | 
| `XXXXX`  | `XXXXXX` | `XXXXXXX` | `XXXXXX` |

  )
}
```

## Customization

> TBC

Reviewer: ```suggestion
No CSS Handles are available yet for the app customization.
```
